Sources/Misc/Concurrency/Purchases+async.swift

```diff
@@ -22,6 +22,14 @@ extension Purchases {
 
     #if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
 
+    func getStorefrontAsync() async -> Storefront? {
+        return await withUnsafeContinuation { continuation in
+            getStorefront { storefrontCountryCode in
+                continuation.resume(returning: storefrontCountryCode)
+            }
+        }
+    }
+
```

Comment:
I believe we don't need this with the existing `func getStorefront() async -> Storefront?` method 🤔